diff --git a/setup.py b/setup.py
index 010e560..e7322bc 100644
--- a/setup.py
+++ b/setup.py
@@ -31,8 +31,8 @@ setup(
     keywords=['meta-learning', 'pytorch', 'few-shot', 'few-shot learning'],
     packages=find_packages(exclude=['data', 'contrib', 'docs', 'tests', 'examples']),
     install_requires=[
-        'torch>=1.4.0,<1.10.0',
-        'torchvision>=0.5.0,<0.11.0',
+        'torch>=1.4.0',
+        'torchvision>=0.5.0',
         'numpy>=1.14.0',
         'Pillow>=7.0.0',
         'h5py',
